src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -57,7 +59,20 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
       i === 0 && 'first',
       i === breadcrumbs.length - 1 && 'last'
     ),
+    'data-background': classNames(
```

Comment:
why are we using the data prop instead of just classname?

Reply:
oh! because data-background is getting generated dynamically based on condition! so we have three diffn shapes based on that we are applying css.

Reply:
Based on EUI's breadcrumb props, it looks like you can add `className` as a prop too. https://eui.elastic.co/v34.6.0/#/navigation/breadcrumbs

```ts
  crumbs = crumbs.map((breadcrumb, i) => ({
    ...breadcrumb,
    'data-test-subj': classNames(
      'breadcrumb',
      breadcrumb['data-test-subj'],
      i === 0 && 'first',
      i === breadcrumbs.length - 1 && 'last'
    ),
    className: classNames(
      'osdHeaderBreadcrumbs',
      i === 0 && 'first',
      i > 0 && i < crumbs.length - 1 && 'middle',
      i === crumbs.length - 1 && 'last'
    ),
  }));
```

Reply:
let me fix that!